log.go

```diff
@@ -400,6 +401,7 @@ func (l *Logger) Print(v ...interface{}) {
 // Arguments are handled in the manner of fmt.Printf.
 func (l *Logger) Printf(format string, v ...interface{}) {
 	if e := l.Debug(); e.Enabled() {
+		e.CallerSkipFrame(1)
```

Comment:
Please inline it the line below.

Reply:
Done. 3326547548fb5fe2a8e6d4db82a7f692a17d6073